ribbon-httpclient/src/main/java/com/netflix/client/http/HttpResponse.java

```diff
@@ -61,10 +60,4 @@ public interface HttpResponse extends IResponse, Closeable {
     public <T> T getEntity(Class<T> type) throws Exception;
 
     public <T> T getEntity(Type type) throws Exception;
-
-    /**
-     * @deprecated use {@link #getEntity(Type)}  
-     */
-    @Deprecated
-    public <T> T getEntity(TypeToken<T> type) throws Exception;
```

Comment:
This is a breaking change because we leaked guava into the public interface, looks like this is deprecated with a replacement so removal of this method will unlock gull guava removal